search-sentiments-map/routes/trends.js

```diff
@@ -46,15 +45,14 @@ async function retrieveGlobalTrends() {
 }
 
 /** 
- * Updates datastore storage of daily search trends and corresponding search results
+ * Updates Datastore storage of daily search trends and corresponding search results
  * for the 46 countries where trends are available using the Google Trends API.
  */
 async function updateDailyTrends() {
-  let countryData = fs.readFileSync('./public/countries-with-trends.json');
-  let countryJson = JSON.parse(countryData);
-
-  var trendsByCountry = [];
-  for (var i = 0; i < countryJson.length; i++) {
+  let countryJson = require('./../public/countries-with-trends.json');
```

Comment:
I think you can make this a const, you won't be changing it.

Reply:
ok